### ISSUE TITLE
Issue when changing the Unity window size (Ver 2.7.0+)

### ISSUE THREAD

**Issue by pavostudio** [RenderHeads/UnityPlugin-AVProVideo#1482]
**Describe the issue**
Please refer to the videos, when I'm using 2.7.0+ and changing the Unity window size, the render view stops and stretches. This issue doesn't happen on 2.6.7, and I have no clue why this happens, any help would be appreciated.

**Setup:**
 - Unity version: 2019.4.40f
 - AVPro Video version (number and edition (trial/core/ultra/enterprise)): 2.7.0-trail/ 2.7.3-core
 - Operating system version: Windows 10/11
 - Project Settings: Run in Background ✔️, Visible in Background ✔️

**Logs**
No error logs.

**Videos**

Video with issue. Using 2.7.0-trail/ 2.7.3-core
https://user-images.githubusercontent.com/35998859/223930168-0463209e-a13b-4ee3-8819-ec62a398b3e3.mp4

Video without issue. Using 2.6.7-trail / 2.6.2-core
https://user-images.githubusercontent.com/35998859/223929071-35ddb707-b324-4173-86d2-30c76b78b603.mp4

**Comment by Chris-RH** [comment]
Hi @pavostudio 
What are the specs of your video? resolutions, codecs etc.
It might be helpful if you could send us a copy of your video you are playing, either here or email unitysupport@renderheads.com

**Comment by Chris-RH** [comment]
So you're saying that just having the mediaplayer component in your project is causing this? Without any files loaded?

**Comment by pavostudio** [comment]
> So you're saying that just having the mediaplayer component in your project is causing this? Without any files loaded?

Yes, and I made another test. This time I used MediaPlayer to play a video as a Live Wallpaper. When I changed the screen resolution, the wallpaper stopped (video, dog, and text with framerate).

Which I can confirm: 
1. It's not related to the video, all videos will have this problem.
2. This issue occurs on 2.7.0+.
3. I didn't change any code besides the AVPro Video package, so I suspect that some changes in the new version are causing this problem.

https://user-images.githubusercontent.com/35998859/227466760-178a44ac-4da4-4e02-b4ca-abf3a1f3524e.mp4
